helm/templates/ssoSecret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sso-express
+  labels: 
+{{ include "connectivity-intake.labels" . | indent 4}}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+type: Opaque
+data:
+  clientSecret: {{ .Values.image.app.clientSecret }}
```

Comment:
```suggestion
  clientSecret: {{ .Values.image.app.clientSecret  | b64enc | quote }}
```

Reply:
yeah just pushed a change to test deployment